scripts/lc-builds/toss4_amdclang.sh

```diff
@@ -73,6 +73,7 @@ cmake \
   -DENABLE_HIP=ON \
   -DENABLE_OPENMP=ON \
   -DENABLE_CUDA=OFF \
+  -DENABLE_BENCHMARKS=ON \
```

Comment:
Should we make this sort of thing an argument you can pass to the script like we do for compiler version, compute architecture, etc?